CHANGELOG.md

```diff
@@ -1,6 +1,39 @@
 # Changelog
 
 ## Unreleased
+### Dashboard
+- Enable ACM/IDM for the dashboard [OP-3674]
+
+#### QA
+Configure the environment variables for the QA ACM/IDM environment:
```

Comment:
We also need to link the new domain name to QA (and remove it from DEV), same for Prod when it's available. Prod probably also needs some DNS config to link it to the server.